commands/verify/verify.go

```diff
@@ -101,9 +103,16 @@ func DoVerifyAccordingConfig() error {
 		return err
 	}
 
+	failFast := e2eConfig.Verify.FailFast
+
 	for idx, v := range e2eConfig.Verify.Cases {
 		if v.GetExpected() == "" {
-			return fmt.Errorf("the expected data file for case[%v] is not specified", idx)
+			errMsg := fmt.Sprintf("the expected data file for case[%v] is not specified", idx)
+			if !failFast {
```

Comment:
I think this is wrong in logic. You should save the error and propagate the errors after all done if not fast fail, otherwise it silently succeeds with just some warning logs, which will just pass the CI process. Also I’d recommend `if failFast { return error` style instead of `if not failFast then continue` style.

Reply:
OK! I will try to fix this !